cmake/get_cmakepp_lang.cmake

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 CMakePP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#[[[ @module
+# .. warning::
+#    This module is only used when building CMakeTest,
+#    and including it automatically pulls in CMakePPLang through
+#    FetchContent.
+#]]
+
+include_guard()
+include(versions)
+
+#[[
+# This function encapsulates the process of getting CMakePPLang using CMake's
+# FetchContent module. When CMaize supports find_or_build for CMake modules this
+# file will be deprecated.
+#]]
+function(get_cmakepp_lang)
+    include(
+        cmakepp_lang/cmakepp_lang
+        OPTIONAL
+        RESULT_VARIABLE cmakepp_lang_found
```

Comment:
I never realized that there were extra possible kwargs for `include()`. Good to know!